### ISSUE TITLE
Text not rendered following latex block

### ISSUE THREAD

**Issue by craigim** [mattermost/mattermost-server#8660]
Per Mattermost guidelines, GitHub issues are for bug reports: http://www.mattermost.org/filing-issues/

For troubleshooting see: http://forum.mattermost.org/.
For feature proposals see: http://www.mattermost.org/feature-requests/

If you've found a bug--something appears unintentional--please report using the following format: 

#### Summary
Text does not render following a latex block

#### Steps to reproduce
Using Mattermost Team Edition version 4.8.1 with postgres database. I left a comment with the following code (not sure how to make the markdown not render here):

---

Note to all:
```latex
\text{mean}(x) = \text{median}(x)
```
only for symmetric distributions (e.g. normal, Cauchy, Voigt). For instance
```latex
\text{mean}(x^2) \approx\left(\frac{9}{7}\right)^3\text{median}(x^2)\approx 2.125 \cdot\text{median}(x^2)
```
because $`x^2`$ is not normally distributed, it is a $`\chi^2_1`$ distribution which is not symmetric. This also holds true for many other distributions that we commonly deal with, such as log-normal and exponential.

---

When I posted the comment back on September 22, 2017 it rendered completely. Now when I view the comment, nothing appears after the first latex block (causing much confusion). Other code (i.e. c++ or matlab) does *not* exhibit this behavior. I've tried creating new comments and get the same results.

#### Expected behavior
After latex block, further text should render

#### Observed behavior (that appears unintentional) 
Screenshot from the Mattermost desktop app (with identifying information blocked out). The view in Firefox is exactly the same. The code I pasted above was copied directly after choosing "edit" on that comment.

![mattermostbugreport](https://user-images.githubusercontent.com/6438424/39143717-b4ed3bc2-46fc-11e8-89b6-f8994ee4c1c8.png)

Note that this behavior does *not* occur after other code blocks, such as c++ or matlab.

**Comment by lindy65** [comment]
Hi @craigim 

Thanks for your feedback,

There have been many improvements to how markdown works.  Please try upgrading to the [latest version](https://about.mattermost.com/download/) to see whether this resolves your issue?

**Comment by craigim** [comment]
I've passed that on to IT and will update the issue when the upgrade has been done.

**Comment by lindy65** [comment]
Thanks @craigim !

Let us know how it goes...

**Comment by SmartHoneybee** [comment]
@lindy65, #8683 sounds similar (and was reproducible on `pre-release.mattermost.com`).

**Comment by hmhealey** [comment]
Hi @craigim ,

What's the plain text for that post like shows up if you try to edit it? You can paste it unformatted on GitHub if you surround it by `~~~`s instead of the regular backquotes

**Comment by craigim** [comment]
`~~~` didn't work for me (code still renders), but here it is with all back quotes replaced with normal single quotes instead (no contractions or possessives, so a search/replace will safely work to get back to valid markup):

---

Note to all:
'''latex
\text{mean}(x) = \text{median}(x)
'''
only for symmetric distributions (e.g. normal, Cauchy, Voigt). For instance
'''latex
\text{mean}(x^2) \approx\left(\frac{9}{7}\right)^3\text{median}(x^2)\approx 2.125 \cdot\text{median}(x^2)
'''
because $'x^2'$ is not normally distributed, it is a $'\chi^2_1'$ distribution which is not symmetric. This also holds true for many other distributions that we commonly deal with, such as log-normal and exponential.

---

**Comment by craigim** [comment]
By the way, the following markup renders just fine (replace single quotes with back quotes):

---

Testing for a code block

'''matlab
x = rand(1);
'''

Can you read this text?

---

**Comment by hmhealey** [comment]
That's really interesting. I've filed a Jira ticket to further investigate that behaviour [here](https://mattermost.atlassian.net/browse/MM-10403).

**Comment by lindy65** [comment]
Thanks @hmhealey :)

@craigim - I'll close this issue off here as the PR has been merged and the fix is on master and will ship with v4.10 on May 16th.
